docs/pages/markdowns/JavaScript_API.md

```diff
@@ -52,6 +52,38 @@ window.mmgisAPI.updateVectorLayer('Waypoints', {
 
 ```
 
+### trimVectorLayer(layerName, keepAfterTime, timePropPath)
+
+This function removes features on a specified layer before a specified time
+
+#### Function parameters
+
+- `layerName` - name of layer to update
+- `keepAfterTime` - absolute time in the format of YYYY-MM-DDThh:mm:ssZ; will keep all features after this time
+- `timePropPath` - name of time property to compare with the time specified by keepAfterTime
+
+The following is an example of how to call the `trimVectorLayer` function:
+
+```javascript
+window.mmgisAPI.trimVectorLayer('Waypoints',  "2021-12-01T15:10:00.000Z", 'time')
+```
+
+### keepLastN(layerName, keepLastN)
+
+This function removes features on a specified layer to keep the specified number of existing features. This function is not aware of time and will only keep the previous N number of features based on the order the features were added to the layer.
+
+#### Function parameters
+
+- `layerName` - name of layer to update
+- `timePropPath` - name of time property to compare with the time specified by keepAfterTime

```

Comment:
`timePropPath` should be removed here.

Reply:
Good catch! I updated in 10df53099c555f93ef25cfe13610f3f9c22fc6da.